tools/clang/test/DXC/fd_strip_dbg.test

```diff
@@ -0,0 +1,8 @@
+// /Fd implies /Qstrip_debug
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Fd %t.fd /Fo %t.dxo
+// RUN:FileCheck --input-file=%t.fd %s --check-prefix=FD | %dxc -dumpbin %t.dxo | FileCheck %s --check-prefix=FO
```

Comment:
Same thing here with the first `|`.

Reply:
Use | will fail for temp file cannot find.